src/content/en/2020/http2.md

```diff
@@ -4,20 +4,530 @@ part_number: IV
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSOkWXtrbMfTLdhlKbBGDjRU3zKbnCQi3iPhfuKaFs5mj4smEzInDCYEnk63gBdgsJ3GFk2gf4FOKCU/pubchart?oid=1779365083&format=interactive",
+  sheets_gid="1457263817",
+  sql_file="cdn_summary.sql"
+)
+}}
+
+Figure 22.13 shows the difference in HTTP/2 and gQUIC adoption when a web site is using a CDN. 70% of pages use HTTP/2 for all 3rd party requests when a CDN is used. Without a CDN, only 25% of pages use HTTP/2 for all third party requests.
+
+Figure 22.14 shows the breakdown of HTTP/2 responses for 3rd party CDNs for mobile requests.
+
+<figure>
+  <table>
+    <thead>
+      <tr>
+        <th scope="colgroup" class="no-wrap">HTTP/2 and gQUIC (%)</th>
+        <th scope="colgroup">Third-party resource CDN</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>100%</td>
+        <td>Bison Grid, Hosting4CDN, MediaCloud, NYI FTW, PageCDN, Pressable CDN, QUIC.cloud, Roast.io, SwiftyCDN, Zycada Networks</td>
+      </tr>
+      <tr>
+        <td>90 - 99%</td>
+        <td>Advanced Hosters CDN, Azion, CDN77, Facebook, GoCache, ImageEngine, KeyCDN, Microsoft Azure, NetDNA, Netlify, Nexcess CDN, Reapleaf, Sirv CDN, Twitter, jsDelivr, section.io, Airee, BunnyCDN, Cloudflare, Erstream, Internap, LeaseWeb CDN, Medianova, Myra Security CDN, NGENIX, OnApp, SFR, StackPath, Sucuri Firewall, TRBCDN, VegaCDN, Yottaa</td>
+      </tr>
+      <tr>
+        <td>70 - 89%</td>
+        <td>Amazon CloudFront, BelugaCDN, CDN, CDN77, Erstream, Fastly, Highwinds, OVH CDN, Yottaa, Edgecast, Myra Security CDN, StackPath, XLabs Security</td>
+      </tr>
+      <tr>
+        <td>20 - 69%</td>
+        <td>Akamai, Aryaka, Google, Limelight, Rackspace, Incapsula, Level 3, Medianova, OnApp, Singular CDN, Vercel, Cachefly, Cedexis, Reflected Networks, Universal CDN, Yunjiasu, CDNetworks</td>
+      </tr>
+      <tr>
+        <td> < 20%</td>
+        <td>Rocket CDN, BO.LT, ChinaCache, KINX CDN, Zenedge, ChinaNetCenter </td>
+      </tr>
+    </tbody>
+  </table>
+<figcaption>{{ figure_link(caption="Percentage of HTTP/2 requests served by the third-party resouce CDNs over mobile", sheets_gid="781660433", sql_file="cdn_detail_by_cdn.sql") }}</figcaption>
+</figure>
+
+## How is HTTP/2 performing ?
+
+Measuring the impact of how a protocol is performing is difficult with the current HTTPArchive approach. It would be really fascinating to be able to quantify the impact of concurrent connections, the effect of packet loss and differrent congestion control mechanisms. To really compare performance each web site would have to be crawled over each protocol over different network conditions. What we can do  instead is to look into the impact on the number of connections a web site uses.
```

Comment:
```suggestion
Measuring the impact of how a protocol is performing is difficult with the current HTTPArchive approach. It would be really fascinating to be able to quantify the impact of concurrent connections, the effect of packet loss and differrent congestion control mechanisms. To really compare performance each web site would have to be crawled over each protocol over different network conditions. What we can do instead is to look into the impact on the number of connections a web site uses.
```